model/user.go

```diff
@@ -111,13 +111,25 @@ type LearningGroup struct {
 	// The slug name of the private channel for this learning group (Channels.Name)
 	ChannelName string `json:"channel_name"`
 
+	// An array of pointers to LearningGroupMember structs for every member of this learning group (channel)
+	Members []*LearningGroupMember `json:"members"`
+
 	// A comma delimited list of the usernames of the members of this learning group (channel)
-	Members *string `json:"members"`
+	MembersDelimitedString *string `json:"members_delimited_string"`
```

Comment:
Do we still need the members delimited string if we have the learning group members?

Reply:
I was going to explain this, but I forgot! The LearningGroups struct represents a row for each result in the getUserLearningGroups query. I wasn't sure how to address that. Does it make more sense to have a struct for the query, and then another struct for the return data?

Reply:
Hmm. So the issue is that one part of the front end (dashboard) wants this delimited string, and another part (recs) wants the members structure? It's a bit ugly, but probably fine in the interests of time.

Reply:
No, they now both use the members structure.

Reply:
Nope it absolutely makes sense to have a struct defined for the query and a different struct for what gets returned by the API. I just didn't want to make you change all that for the last PR.
Those really are 2 different things and they should be defined as such.

Reply:
Really? It will only have 1 different field, and the rest will be duplicates.

Reply:
Maybe it's better to use this technique. After we process MembersDelimitedString, set it to nil, and set the omit empty on it so it doesn't get passed to the front-end:
https://stackoverflow.com/questions/24216510/empty-or-not-required-struct-fields-in-golang

Reply:
No, I still say that the struct returned by the API is a different entity from the struct returned internally by the query you wrote and they should be defined separately, and this is just the 1st crack in why they are different things, more will probably arise in the future. I was close to asking for the 2 struct definitions when this code was initially written but let it go since they will identical at that time.

The technique you're citing is not appropriate in my mind for this.

Reply:
Ok fair point. I think using pointers for null values in the struct is only required if the struct is a sql query result row...so it removes the need to use them in the return struct. I will make that adjustment.